statemachine/statemachine.py

```diff
@@ -26,11 +27,21 @@ class StateMachine(metaclass=StateMachineMetaclass):
     states = []  # type: List[State]
     states_map = {}  # type: Dict[Any, State]
 
-    def __init__(self, model=None, state_field="state", start_value=None):
+    def __init__(
+        self,
+        model: Any = None,
+        state_field: str = "state",
+        start_value: Any = None,
+        queued: bool = True,
+    ):
         self.model = model if model else Model()
         self.state_field = state_field
         self.start_value = start_value
+        self.__queued = queued
+        self.__processing: bool = False
+        self._external_queue: deque = deque()
 
+        assert hasattr(self, "_abstract")
```

Comment:
author's note: Used to silence mypy complaing about `_abstract` not defined. This attr is added using meta-programing on the metaclass.

> `statemachine/statemachine.py:44: error: "StateMachine" has no attribute "_abstract"  [attr-defined]`